docs/reference/release-notes/6.7.asciidoc

```diff
@@ -287,13 +287,14 @@ License::
 * Handle malformed license signatures {pull}37137[#37137] (issue: {issue}35340[#35340])
 
 Machine Learning::
-* [ML] Allow stop unassigned datafeed and relax unset upgrade mode wait {pull}39034[#39034]
+* Allow stop unassigned datafeed and relax unset upgrade mode wait {pull}39034[#39034]
 * Move ML Optimistic Concurrency Control to Seq No {pull}38278[#38278] (issues: {issue}10708[#10708], {issue}36148[#36148])
-* [ML] Add explanation so far to file structure finder exceptions {pull}38191[#38191] (issue: {issue}29821[#29821])
-* ML: Add upgrade mode docs, hlrc, and fix bug {pull}37942[#37942]
-* [ML] Tighten up use of aliases rather than concrete indices {pull}37874[#37874]
-* ML: Add support for single bucket aggs in Datafeeds {pull}37544[#37544] (issue: {issue}36838[#36838])
-* [ML] Migrate unallocated jobs and datafeeds {pull}37536[#37536] (issues: {issue}32905[#32905], {issue}36810[#36810], {issue}37430[#37430], {issue}37656[#37656])
+* Add explanation so far to file structure finder exceptions {pull}38191[#38191]
```

Comment:
This one went into 6.6.1.  Is it usual to also include it in a subsequent release?

Reply:
No, good catch, thanks!